src/rpc/service_eth.go

```diff
@@ -165,10 +165,32 @@ func (service *EthService) SendRawTransaction(ctx context.Context, s string) (*c
 	if err := tx.UnmarshalBinary(b); err != nil {
 		return nil, &EncodingError{StatusCode: -32602, Err: err}
 	}
+
 	txHash := tx.Hash()
-	txFromAddress, err := utils.SenderAddress(tx)
+	fromAddress, err := utils.SenderAddress(tx)
+	if err != nil {
+		return nil, &EncodingError{StatusCode: -32602, Err: err}
+	}
+
+	accountNonce, err := service.Processor.Client.NonceAt(ctx, fromAddress, nil)
+	if err != nil {
+		return nil, &EncodingError{StatusCode: -32602, Err: err}
+	}
+
+	if accountNonce != tx.Nonce() {
```

Comment:
I'm not entirely sure, if this is actually correct. It should be `accountNonce > tx.Nonce`. IOW it should be allowed to send a transaction with nonce > TransactionCount

Reply:
Fixed to allow for transactions with higher nonces to still be sent by the user.